__tests__/app.test.js

```diff
@@ -120,6 +120,77 @@ describe("GET/api/articles", () => {
   });
 });
 
+describe("GET/api/articles (topic query)", () => {
+  it("200 status - get articles filtered by topic", () => {
+    return request(app)
+      .get("/api/articles/?topic=mitch")
```

Comment:
the url format should be "/api/articles?topic=mitch"

Dont need the / before the ?